source/code-snippets/usage-examples/findOne.ts

```diff
@@ -0,0 +1,40 @@
+import { FindOptions, MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
+
+const client = new MongoClient(uri);
+
+type imdb = {
+  rating: number;
+  votes: number;
+  id: number;
+};
```

Comment:
```suggestion
interface IMDB {
  rating: number;
  votes: number;
  id: number;
}
```

Nothing wrong with using `type` here but might as well be consistent and use an `interface`. Also regardless is common that types start with a capital letter.

Reply:
Sounds good, will try to remember to capitalize types and be consistent with `type` and `interface` usage throughout these usage examples.